src/components/enterprise-page/EnterprisePage.test.jsx

```diff
@@ -24,48 +27,89 @@ describe('<EnterprisePage />', () => {
   beforeEach(() => {
     jest.clearAllMocks();
   });
+  const queryClient = new QueryClient({
+    queryCache: new QueryCache({
+      onError: queryCacheOnErrorHandler,
+    }),
+  });
+  const QueryClientProviderWrapper = ({ children }) => (
+    <QueryClientProvider client={queryClient}>
+      {children}
+    </QueryClientProvider>
+  );
+
+  const EnterprisePageWrapper = ({ children, appContextValue }) => (
```

Comment:
nit: given majority of test cases do rely on the same default value, it might still be worth creating a default variable representing the standard `appContextValue` you'd expect so that only the test case(s) with the exception to the default value needs to pass `appContextValue`; that is, all the other test cases wouldn't need to pass `appContextValue`.